mycroft/client/wifisetup/main.py

```diff
@@ -38,6 +42,29 @@
 
 LOG = getLogger("WiFiClient")
 
+SCRIPT_DIR = dirname(realpath(__file__))
+
+
+def var_dump(var, prefix=''):
+    my_type = '[' + var.__class__.__name__ + '(' + str(len(var)) + ')]:'
+    print(prefix, my_type)
+    prefix += '    '
+    for i in var:
+        if type(i) in (list, tuple, dict, set):
+            var_dump(i, prefix)
+        else:
+            if isinstance(var, dict):
+                print(prefix, i, ': (', var[i].__class__.__name__, ')', var[i])
+            else:
+                print(prefix, '(', i.__class__.__name__, ')', i)
+
+
+def cli_no_output(*args):
```

Comment:
Since I don't see where I can respond to the rebase commend above, I'll do it here:

This code needs to be applied to Master as soon as possible.  I'm afraid we cannot wait until we are ready to release the Dev branch on everyone because:
1) This process is particularly needed now after the package change that wiped out the wpa_supplicant file.
2) This functionality makes doing demos WAY easier
3) This process is the most critical thing that has to be in place on the initial device image, and the day we need to make that is rapidly approaching.  Without it the unit is useless to users who don't use Ethernet cables.  Everything else can be upgraded once the unit is on the network.